templates/travis/.travis/before_install.sh.j2

```diff
@@ -51,6 +51,15 @@ black --check --diff .
 # Lint code.
 flake8 --config flake8.cfg
 
+{%- if docker_fixtures -%}
+# run fixtures container and update pulp-smash config
+docker run -d -p 8000:80 quay.io/pulp/pulp-fixtures:latest
```

Comment:
:heart:

Reply:
Yeah, I agree with this approach for launching the container. It's the least work vs putting it in k8s.